### PR TITLE
Project_id filtered

### PR DESCRIPTION
**Pull request by ianlcz**

### CODE REVIEW

File: utils/database.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import os
 
 from models.file import File
+from models.ownership import Ownership
+from models.version import Version
 from utils.proglang import guess_programing_language
 
 def save_file_if_not_found(session, file_path):
-    file = session.query(File).filter(File.path == file_path).first()
```
Comment (bbalet) [inline utils/database.py:7]: This code is not related to the ownership metric, which is codemaat related

File: connectors/codemaat.py [hunk 2]
```diff
@@ -172,7 +172,10 @@ def ownership_patterns(self, git_log_file):
         # Insert the patterns of ownership into the database
         for index, row in df.iterrows():
             # Create the author if it doesn't exist
-            author = self.session.query(Author).filter(Author.name == row['author']).first()
+            author = self.session.query(Author, Ownership) \
+                    .join(Ownership, Author.author_id == Ownership.author_id) \
+                    .filter(Author.name == row['author']) \
+                    .filter(Ownership.version_id == Version.version_id).first()
```
Comment (bbalet) [inline connectors/codemaat.py:178]: This can't wok as Version.version_id is used as a kind of unset static field

File: utils/database.py [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+    file = session.query(File, Ownership) \
+        .join(Ownership, File.file_id == Ownership.file_id) \
+        .filter(Ownership.version_id == Version.version_id) \
```
[the remark this answers is not in the document]
Comment (ianlcz) [reply utils/database.py:11]: File model hasn't no version_id attribut to filtered by project.
Then I joined File and Ownership with common attribut `file_id`.
Then I wanted to filter Ownership with Version but in utils.database we have not version/project properties.

File: connectors/codemaat.py (hunk 2 above)
Comment (ianlcz) [reply connectors/codemaat.py:178]: I fix it
